maquetado/scss/details.scss

```diff
@@ -83,3 +82,69 @@
   font-size: $font-normal;
   margin-bottom: 110px;
 }
+
+.back-container {
+  display: flex;
+  height: 100%;
+  justify-content: flex-start;
+  max-height: 416px;
+  max-width: 900px;
+  width: 100%;
+}
+
+.back-arrow-down {
+  display: none;
+}
+
+@media only screen and (max-width: 1024px) {
+  .book-card {
+    flex-flow: column;
+    justify-content: center;
+    max-height: 900px;
+    max-width: 416px;
+  
+    .book {
+      .book-title {
+        padding: 0 8px 12px 0;
+      }
+    }
+  
+    .book-cover {
+      padding: 24px;
+      margin-right: 0;
+  
+      .book-badge {
+        max-height: $badge-small-h;
+        max-width: $badge-small-w;
+        height: 100%;
+        width: 100%;
+        right: 14px;
+        top: 4px;
+      }
+    }
+  }
+
+  .back-arrow-down::before {
+    content: url('assets/back-arrow-20.png');
+    padding-right: 12px;
+  }
+  
+  .back-arrow {
+    display: none;
+  }
+
+  .back-arrow-down {
+    display: flex;
+    font-size: $font-normal;
+    margin-top: 0;
+    margin-bottom: 0;
```

Comment:
Do you need to set a margin in 0? Because in desktop the margin is not set.

Reply:
Done!